### DISCUSSION TITLE
[Feature] server statistic | Show the amount of Gib allocated to the user

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#7582] [Feature Request]
### The feature

It would be a nice addition to see the total Gib number defined for the user on the server statistic page.

before:
![image](https://github.com/immich-app/immich/assets/51673860/e98a2997-7ef3-4560-bd58-29627158ea84)

after:
![image](https://github.com/immich-app/immich/assets/51673860/2532b8cc-2088-4b77-b629-38ee58fabc02)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by samholton** [comment]
Added in https://github.com/immich-app/immich/pull/7591

**Reply by aviv926** [reply]
Thanks!
